app/services/opening_projects_csv_exporter.rb

```diff
@@ -72,4 +72,8 @@ def call
   private def fetch_mp_details(project)
     Api::MembersApi::Client.new.member_for_constituency(project.establishment.parliamentary_constituency)
   end
+
+  private def format_date(date)
```

Comment:
Is it worth putting this date format into the config and using `to_formatted_s(:csv_export)`?